compareAll.sh

```diff
@@ -49,8 +49,10 @@ IFS=',' read -r -a tags <<< $3
 erasarg=$1
 channelsarg=$2
 tagsarg=$3
+conditional_arg=$4
 
 if [[ $eras == "all" ]]; then
```

Comment:
eras->erasarg

Reply:
Done

Reply:
also, in line 74, all sould be accepted as a valid era.
`if [[ ! "2016 2017 2018" =~ ${era} ]]; then` 
->
`if [[ ! "2016 2017 2018 all" =~ ${era} ]]; then`

Reply:
Nicht nötig wegen [[ "" = $( echo ${eras} ) ]] && eras=("2016" "2017" "2018") erasarg="2016,2017,2018"